package/network/services/hostapd/files/hostapd.sh

```diff
@@ -574,6 +596,11 @@ hostapd_set_bss_options() {
 			append bss_conf "macaddr_acl=0" "$N"
 			append bss_conf "deny_mac_file=$_macfile" "$N"
 		;;
+		radius)
+			append bss_conf "macaddr_acl=2" "$N"
+			# radius server can be used to assign dynamic VLAN based on MAC
+			vlan_possible=1
+		;;
```

Comment:
You should clear there `_macfile` variable, like few lines below for `*)`... `_macfile=""`